src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -100,54 +96,62 @@ Each option for incident preference has its own advantages. You can select any o
-</figcaption>
-
 To change the incident preference for the selected policy:
 
 1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, then click **Policies**, and then (**select a policy)**.
 2. Click **Incident preference**, select your choice of available [options](#preference-options), and then save.
 
 Repeat these steps for each policy as needed.
 
-## Example: By policy (default) [#preference-policy]
+![A screenshot showing where the incident preference button is inside New Relic One's Alerts user interface.](./images/incident-preference-UI-1.png "A screenshot showing where the incident preference button is inside New Relic One's Alerts user interface.")
 
-![Alert policy incident preferences](./images/example-policy-incident.png "Alert policy incident preferences")
+![A screenshot showing how to change your incident preferences on New Relic One's Alerts user interface.](./images/incident-preferences.png "A screenshot showing how to change your incident preferences on New Relic One's Alerts user interface.")
 
 <figcaption>
-  When you select the default (**By policy**) as your incident preference, it will group all condition violations into the same incident.
+  **[one.newrelic.com](https://one.newrelic.com) > Alert & AI > Policies > (select a policy):** The selected policy page shows how alerts rolls up incidents for alert notifications and UI details. (Default is **By policy**). To choose a different option for this policy, click **Incident preference**.
 </figcaption>
 
-By default, a single incident record will be created for each alert policy. If additional conditions are violated by any target after the incident opens, the violations are added to the same incident. Only one incident appears on your **Incidents** indexes for the selected policy.
 
-Selecting the **By policy** preference is the most general option. All condition violations for every target will be accumulated into a single incident record.
 
-## Example: By condition [#preference-condition]
+## By policy (default) [#preference-policy]
+
+![A diagram showing how the By policy incident preference works](./images/by-policy.svg "A diagram showing how the By policy incident preference works")
 
-![Alert policy incident preferences](./images/example-condition-incident.png "Alert policy incident preferences")
+<figcaption>
+  When you select the default (**By policy**) as your incident preference, it will group all condition violations into the same incident.
+</figcaption>
+
+If additional conditions are violated by any target after the incident opens, the violations are added to the same incident. Only one incident appears on your Incidents indexes for the selected policy.
```

Comment:
It'd be grand if you could make that first sentence active and not passive, like it is.

Also, I think it should be "Incidents index"